histomicstk/ComputeHaralickFeatures.py

```diff
@@ -0,0 +1,199 @@
+import numpy as np
+import pandas as pd
+import skimage.feature
+from skimage.measure import regionprops
+
+
+def ComputeHaralickFeatures(I, Label, Offsets=[[0, 1]], NumLevels=256,
+                            MaxGray=255, MinGray=0):
```

Comment:
Binning the gray levels provides some robustness against noise or bias in intensity from one image to another. Maybe go with ~10 gray levels by default?